pkgs/development/python-modules/plyfile/default.nix

```
@@ -11,6 +11,8 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [ numpy ];
+  buildInputs = [ pdm-pep517 ];
```

probably doesn't make a massive difference, but I think nativeBuildInputs would be more appropriate.
```suggestion
  nativeBuildInputs = [ pdm-pep517 ];
```